src/Controller/UserController.php

```diff
@@ -17,6 +18,14 @@
  */
 class UserController extends AbstractController
 {
+
+    private $emailService;
```

Comment:
On top of class variables like this, a phpdoc comment `@var EmailService` doesn't eat bread (lol)

Reply:
Done